plugins/out_s3/s3.c

```diff
@@ -119,6 +120,38 @@ static char *mock_error_response(char *error_env_var)
     return NULL;
 }
 
+static void s3_retry_warn (struct flb_s3 *ctx, char tag, char input_name, time_t create_time, bool less_than_limit){
+    struct tm now_time;
+    char create_time_str[20];
+    strftime(create_time, 20, "%Y-%m-%d %H:%M:%S", localtime_r(&create_time, &now_time));
```

Comment:
One Fluent Bit code practice/standard (which I sometimes forget to follow myself) is that we never call functions within functions. So the localtime_r call needs to be the line above.